lmfdb/classical_modular_forms/web_newform.py

```diff
@@ -119,6 +120,77 @@ def td_wrapr(val):
 def parity_text(val):
     return 'odd' if val == -1 else 'even'
 
+def display_hecke_polys(form_label, num_disp = 5):
+    """
+    Display a table of the characteristic polynomials of the Hecke operators
+    for small primes. The number of primes presented by default is 5, although
+    there is a "show more" / "show less" button to display all primes up to 97.
+    The code for the table wrapping, scrolling etc. is common with many others
+    and should be eventually replaced by a call to a single class/function with
+    some parameters.
+
+    INPUT:
+
+    - ``form_label`` - a string, the label of the newform
+    - ``num_disp`` - an integer, the number of characteristic polynomials to display by default.
+    """
+    #from time import clock
```

Comment:
Does LMFDB have a position on commented out code? (Here and below.) I couldn't find this addressed in the Developer's guide. 

In general I feel that this should be avoided, although in this instance I can see how someone *could* make a case to keep this profiling code commented out

Reply:
I would just remove it

Reply:
Yep, that was also my inclination. They'll be gone in the next commit.